FWCore/Skeletons/scripts/mkdqmedanalyzer

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+#pylint: disable-msg=
+"""
+File       : mkdqmedanlzr
+Author     : Marco Rovere
```

Comment:
I doubt you meant to be the autor for ever single one of these :)

Reply:
Ah, I miss read, this is not the skeleton itself but is the generator. Sorry about that.